src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindOperation.java

```diff
@@ -59,14 +62,16 @@ public record FindOperation(
    * @param projection projections, see FindOperation#projection
    * @param readType type of the read
    * @param objectMapper object mapper to use
+   * @param includeSortVector include sort vector in the response
    * @return FindOperation for a single document unsorted find
    */
   public static FindOperation unsortedSingle(
       CommandContext commandContext,
       LogicalExpression logicalExpression,
       DocumentProjector projection,
       ReadType readType,
-      ObjectMapper objectMapper) {
+      ObjectMapper objectMapper,
+      boolean includeSortVector) {
```

Comment:
Do we still need this passed even though this is unsorted? Or in general, not just for "vsearch" case? I guess spec allows specifying `includeSortVector` for all find()/findOne() cases so perhaps that makes sense.

Reply:
Looks like spec explicitly allows use without actual ANN so this makes sense.